xbmc/cores/VideoPlayer/Process/ProcessInfo.h

```diff
@@ -110,4 +114,5 @@ class CProcessInfo
   // player states
   CCriticalSection m_stateSection;
   bool m_stateSeeking;
+  std::atomic_int m_levelVQ;
```

Comment:
is CProcessInfo a private interface? I assumed otherwise because PAPlayer uses it